bin/build

```diff
@@ -1,13 +1,29 @@
 #!/usr/bin/env bash
 
+DEBUG=("$1" == "debug")
+
 set -e
 
+cd $(dirname $0)/..
+
 . bin/build_utils
 
 FULL_VERSION_TAG="$(full_version_tag)"
 
 echo "---"
 
+if [ $DEBUG ]; then
+
+  echo "Building secrets-provider-debug Docker image"
+
+  docker build \
+    --tag "secrets-provider-debug:latest" \
```

Comment:
let's maintain the name `secrets-provider-for-k8s` so in this case it can be `secrets-provider-for-k8s-debug`. same goes for the target name.